client/a8c-for-agencies/sections/referrals/referrals-list/index.tsx

```diff
@@ -48,62 +48,24 @@ export default function ReferralList( {
 		() =>
 			dataViewsState.selectedItem || ! isDesktop
 				? [
-						// Show the client column as a button on mobile
 						{
 							id: 'client',
 							label: translate( 'Client' ).toUpperCase(),
 							getValue: () => '-',
 							render: ( { item }: { item: Referral } ): ReactNode => (
-								<Button
-									className="view-details-button client-email-button"
-									data-client-id={ item.client.id }
-									onClick={ () => openSitePreviewPane( item ) }
-									borderless
-								>
-									{ item.client.email }
-								</Button>
+								<span className="a4a-referrals-client">{ item.client.email }</span>
```

Comment:
I wanted to make it look the same as before, so added this class to target it. Is this the proper naming structure?